server/services/TransferService.js

```diff
@@ -1,15 +1,18 @@
+const log = require('loglevel');
 const Session = require('../infra/database/Session');
 const Transfer = require('../models/Transfer');
 const HttpError = require('../utils/HttpError');
 const WalletService = require('./WalletService');
 const TokenService = require('./TokenService');
 const TransferEnums = require('../utils/transfer-enum');
+const Event = require('../models/Event');
 
 class TransferService {
   constructor() {
     this._session = new Session();
     this._transfer = new Transfer(this._session);
     this._walletService = new WalletService();
+    this._event = new Event(this._session);
```

Comment:
I'd rather we expose the log function via the EventService and import the EventService instead of the Model across other services

Reply:
Added the log function to EventService and changed all logEvent function calls to EventService method